### ISSUE TITLE
An inquiry about the hyper parameter 'll_weight'

### ISSUE THREAD

**Issue by Amayama** [Yu-Fangxu/FoR#1]
I notice the you had a hyper parameter called `ll_weight` in each `run.sh`, and I cannot find the corresponding parameter in the repo and the paper. Can you shall how to select the parameter?

**Comment by Yu-Fangxu** [comment]
Hi Amayama,
ll_weight is the coefficient of \lambda in the paper, you can find it in Section 4.2, Reward Design.
ll_weight is used in lightning_module_selection.py
I selected it by choosing the best score on the training data, as you can see in Figure 3(a) in the paper

**Comment by Amayama** [comment]
Great thanks for the kind reply!
